### PR TITLE
Creating a file for Levittown PA

### LINKED ISSUE #22: Github rdahlM3's Hometown Profile
Tell the world fun things to do in my hometown
## Levittown, PA
- [x] create a branch
- [x] create a file
- [x] commit changes
- [x] open pull request
- [x] have discussion
- [x] merge pull request

### PR DESCRIPTION
**Pull request by rdahlM3**
This Pull Request resolves #22

### CODE REVIEW

File: places/levittown-pa.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Levittown, PA
+
```
Comment (rohitssurve) [inline places/levittown-pa.md:2]: @rdahlM3 Thanks

**Comment by jmaciorowski** [comment]
Your code is bad

File: places/levittown-pa.md (hunk 1 above)
Comment (jmaciorowski) [inline places/levittown-pa.md:2]: Your code is bad!

**Comment by jmaciorowski** [comment]
Fantastic

File: places/levittown-pa.md (hunk 1 above)
Comment (rohitssurve) [inline places/levittown-pa.md:2]: :shipit:

**Pull request merged**